templates/web-application/README.md

```diff
@@ -52,6 +52,22 @@ Run linting on {{PACKAGE-NAME}} files.
 pnpm lint
 ```
 
+### storybook
+
+Runs storybook at the address [http://localhost:6006](http://localhost:6006).
+
+```bash
+pnpm storybook
+```
+
+### chromatic
+
+This requires a [Chromatic](https://www.chromatic.com/) account and a Chromatic token in the `CHROMATIC_TOKEN` environment variable.
+Make sure to also specify the `CHROMATIC_PROJECT_TOKEN` either in an environment variable or via the `--project-token` CLI flag.
```

Comment:
Should the template include a default pipeline to execute Chromatic on the CI?

Reply:
Yes! i beleive this is the next steps for the template. I would expect people using the pipeline more than the cmd line